Installing linter files

marcometz opened the pull request:
rake Task to copy Linter files to app root

Commit:
Installing linter files

rake Task to copy Linter files to app root

donni106 commented:
Wenn ich hier lese denke ich, du hättest nur einen rake task erstellt.

marcometz commented:
habe ich im Grunde doch eigentlich auch :-(

donni106 commented on Gemfile:
```diff
@@ -1,4 +1,7 @@
 
-# Specify your gem's dependencies in ikusei-linters.gemspec
-gemspec
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
```
Was ist denn das für ein Trick?

marcometz replied:
Sicherstellung das die Quellen per https eingebunden werden: https://stackoverflow.com/questions/41454333/meaning-of-new-block-git-sourcegithub-in-gemfile

donni106 commented on lib/tasks/linters.rake:
```diff
@@ -0,0 +1,27 @@
+    target = File.join(Rails.root, ".scss-lint.yml")
+    FileUtils.cp source, target
+  end
+end
```
Aus den drei Bodys könnte eine Methode entstehen die den Namen ohne Punkt reinkriegt und dann mit Punkt wohinkopiert?! Keine Pflicht natürlich, nur ne Schönheitsoperation.

donni106 reviewed (changes requested):
Code Climate hat noch was zu melden.
Und für das Projekt selbst scheinen die Linter nicht zu greifen:
https://codeclimate.com/repos/57d18afa46cfc50aa1000c33/compare/with_installer

donni106 commented:
Das gem ist bei Rubygems: https://rubygems.org/gems/ikusei_linters
Ist dann dafür noch was zu tun? Vllt. sollte es dort nicht mehr sein denn, wenn kein ikusei mehr im Namen ist?

donni106 committed:
Merge branch 'update-eslint-configuration' into with_installer

donni106 commented:
Habe eslint Konfiguration hier aktualisiert (und im master).

Commit:
Rails/HttpStatus not Working

donni106 committed:
Update rubocop.yml

- there are new configs available with newer RuboCop versions
- the changes are based on the default RuboCop configs:
https://github.com/rubocop-hq/rubocop/blob/master/config/default.yml

donni106 committed:
Update rubocop version in gemspec to ~> 0.58.0

donni106 commented:
Was machen wir hiermit?

acousma82 commented:
Keine Ahnung ? Mergen ? Und dann fange ich an rubocp rspec zu  integrieren ?

donni106 commented:
Ich weiß leider nicht, ob noch etwas vor dem Merge fehlt.

acousma82 commented:
Ich klann es nicht überprüfen, denn ich weiß nicht was im Detail fehlt @marcometz . Schau Du es Dir doch mal bitte an.
